frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -113,19 +140,15 @@ export const InfoContent: React.FC<IInfoContent> = ({
           <ThreeColumnItem leftSideLabel={'Longitude'} rightSideItem={propertyInfo?.longitude} />
         </OuterRow>
       </ListGroup>
-      {(keycloak.hasAgency(propertyInfo?.agencyId as number) ||
-        keycloak.hasClaim(Claims.ADMIN_PROPERTIES)) && (
-        <>
-          {propertyTypeId === PropertyTypes.PARCEL && (
-            <ParcelAttributes
-              addAssociatedBuildingLink={addAssociatedBuildingLink}
-              parcelInfo={propertyInfo as IParcel}
-            />
-          )}
-          {propertyTypeId === PropertyTypes.BUILDING && (
-            <BuildingAttributes buildingInfo={propertyInfo as IBuilding} />
-          )}
-        </>
+      {propertyTypeId === PropertyTypes.PARCEL && (
+        <ParcelAttributes
+          addAssociatedBuildingLink={addAssociatedBuildingLink}
+          parcelInfo={propertyInfo as IParcel}
+          canViewDetails={canViewProperty}
+        />
+      )}
+      {propertyTypeId === PropertyTypes.BUILDING && (
+        <BuildingAttributes buildingInfo={propertyInfo as IBuilding} />
```

Comment:
It looks like both ParcelAttributes and BuildingAttributes used to be hidden by a conditional render if they didn't have the agency/claim. Now you are passing canViewProperty to ParcelAttributes, which I like, but not passing that to the BuildingAttributes?

Reply:
I'm just adding it now as I'm working on the building slide out, it'll be in my next PR